### PR TITLE
Improve compatibility with composer install --no-dev

### PR DESCRIPTION
**Pull request by jkrzefski**

### CODE REVIEW

File: config/bundles.php [hunk 1]
```diff
@@ -5,26 +5,23 @@
...
-    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
     Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true, 'test' => true],
     Enqueue\Bundle\EnqueueBundle::class => ['all' => true],
     Enqueue\MessengerAdapter\Bundle\EnqueueAdapterBundle::class => ['all' => true],
     Shopware\Core\Framework\Framework::class => ['all' => true],
     Shopware\Core\System\System::class => ['all' => true],
     Shopware\Core\Content\Content::class => ['all' => true],
     Shopware\Core\Checkout\Checkout::class => ['all' => true],
-    Shopware\Core\Profiling\Profiling::class => ['dev' => true],
+    Shopware\Core\Maintenance\Maintenance::class => ['all' => true],
```
Comment (keulinho) [inline config/bundles.php]: the composer version specifies that `"shopware/core": "~v6.4.0"` is required, so we can't expect that the Maintenance and DevOps bundles are present in all installations i guess. 

Please bring back the class exists checks for both cases, maybe you can add a `@deprecated tag:v6.5.0` annotation so we will remove the class exists checks on the next major version

**Review by keulinho — CHANGES_REQUESTED** [review]

File: config/bundles.php (hunk 1 above)
Comment (jkrzefski) [reply config/bundles.php]: Done. Thank you.

**Comment by tinect** [comment]
@keulinho any progress? 🤔

**Comment by tinect** [comment]
@keulinho any process? 🤔

**Comment by keulinho** [comment]
Hey @jkrzefski we finally merged your PR, thx for your contribution! :blue_heart: :tada:

**Pull request closed without merging**